test/Controller/MiddlewareControllerTest.php

```diff
@@ -1,135 +0,0 @@
-<?php
```

Comment:
I think we should drop these in 4.x, not 3.x 🤔

Reply:
They are dropped in 4.x. 
hey are also not usable in PHP 8 and they prevent proper php version resolution.

They were requiring PHP below 8, but again I see that "merge" commit: https://github.com/laminas/laminas-mvc/commit/ad9267a938e08af19a1e1ef43122a2ba9f807291#diff-388c804d6485a66f01b14f8f443ca49bfe180c14bb340fac584ff613129f6363L23

Reply:
Full reasons for dropping it:
- This requires dependencies not installable on any supported PHP versions. Using it is already on own risk.
- It interferes with dependency resolution because platform ignores are required. Renovate can not operate on this.
- Optional stratigility version supported here was tested on up to PHP 7.2.
- This was deprecated since laminas-mvc 3.2
- This functionality is entirely replaced by laminas/laminas-mvc-middleware 1.0. Replacement does not use tested code and properly declares dependencies.
- No changes will be accepted for either of the pieces tested by removed tests.